src/hooks/useCurrencies.ts

```diff
@@ -102,7 +104,8 @@ export const useCurrencies = () => {
 
       const convertedCurrency = convertCurrency(
         price.currency as CurrencyCode,
-        targetCurrency
+        targetCurrency,
+        amount
       );
```

Comment:
Seeing as we are returning early here, the `amount` or `multiplier` will not be applied if the source and target currencies are the same